### DISCUSSION TITLE
Best practices for combing datasets

### DISCUSSION THREAD

**Discussion by lawsonvt** [satijalab/seurat#6100] [Q&A]
Hi,

I am analyzing two publicly available datasets and want to combine them for an analysis. The first one is a "healthy" dataset that has multiple donors and the second one is a "diseased" dataset that has multiple biological duplicates from the same diseased tissue. Both datasets stem from the same type of tissue and I have aligned the raw fastq files from both datasets to the same reference genome with the same alevin-fry workflow.

What is the best way to combine this data, so I can compare healthy vs disease while still trying to account for donor variability and batch effects stemming from the fact that they were generated by two different labs? Read all of the data into one Seurat object and use something like Harmony? Or leave them separate, smooth out any internal batch effects and THEN integrate them?

Thanks for any help with this,

~ Mark

**Comment by f6v** [comment]
I guess you're aware of Seurat data integration (they have tutorials [here](https://satijalab.org/seurat/articles/integration_introduction.html)).

What I usually do is try several integration tools (Seurat, BBKNN) and compare the results. Also try running a full workflow on uncorrected data to see if there's a strong batch effect in the first place. If different tools agree, you should go with whatever you're comfortable with. 

Be aware of using different assays for for different things, i.e. "integrated" only for clustering and UMAP. Alternatively, there's a new [way](https://satijalab.org/seurat/articles/sctransform_v2_vignette.html) of using the SCT assay for everything.

**Reply by lawsonvt** [reply]
Thanks for the tips! Yes, I have been looking through the tutorials (I have also tried out [this one](https://satijalab.org/seurat/articles/integration_large_datasets.html), since the CCA algorithm was taking far too long) and trying out different approaches. I will continue to experiment.
